eZ/Publish/Core/Persistence/Legacy/Tests/Content/MapperTest.php

```diff
@@ -251,6 +323,30 @@ public function testExtractContentFromRowsMultipleVersions()
         );
     }
 
+    /**
+     * @param string[] $fields
+     */
+    private function getFieldRegistry(
+        array $fields = [],
```

Comment:
naming: the expected contents of that variable is not self-explanatory enough:
How about:
```suggestion
        array $fieldTypeIdentifiers = [],
```